### PR TITLE
adding temporary Queen Teuta portrait and adding to credits

### PR DESCRIPTION
**Pull request by jcmundy**
This should only affect the credits message, and add one portrait image.  Thanks!

### CODE REVIEW

**Comment by jcmundy** [comment]
Okay, since I created this - it has added my two new commits.  Those include my upgrading Unity, and the addition of two new city model prefabs.  Might be worth testing?  Somehow?

File: Assets/Resources/credits_writeup.txt [hunk 1]
```diff
@@ -4,67 +4,211 @@
 
 CREDITS & ACKNOWLEDGMENTS
```
Comment (kddressel) [inline Assets/Resources/credits_writeup.txt:6]: i think we should just delete this file. i can do it after the merge

**Comment by jcmundy** [comment]
Okay, I wasn't sure which of the two credits txt files was used, so I just updated both.  I knew one was outdated.

File: Assets/Resources/credits_writeup.txt (hunk 1 above)
Comment (jcmundy) [reply Assets/Resources/credits_writeup.txt:6]: Oh, I replied in the general PR.

**Comment by kddressel** [comment]
just tested locally and the new model is there and looks great!

**Comment by kddressel** [comment]
merging, and then i'm fixing these minor issues after the merge:
* remove unity test framework and rider to fix unity 2019.4.9 bug where packages-lock.json keeps changing. see https://forum.unity.com/threads/packages-lock-json.912602/
* assigning missing click reference on the new Pherai model
* deleting unused duplicate credits file

**Pull request merged**